examples/evm/nft-linker/NftLinkerProxy.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { Proxy } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/upgradable/Proxy.sol';
+
+contract NftLinkerProxy is Proxy {
+    bytes32 internal constant CONTRACT_ID = keccak256('nft-linker');
+
+    constructor(address implementationAddress, address owner, bytes memory setupParams) Proxy(implementationAddress, owner, setupParams) {}
+
+    // slither-disable-next-line dead-code
```

Comment:
```suggestion
```
add the same slither/solhint config to this repo?

Reply:
added solhint/slither configs to ignore these warnings